pkg/telemetery/settings.go

```diff
@@ -0,0 +1,13 @@
+package telemetery
+
+import (
+	"github.com/jaegertracing/jaeger/pkg/jtracer"
+	"go.opentelemetry.io/otel/metric"
+	"go.uber.org/zap"
+)
+
+type TelemeterySetting struct {
```

Comment:
the package name is already `telemetry`, so the type will be called `telemetry.Settings`
```suggestion
type Setting struct {
```